test-data/src/test/resources/org/openmrs/module/fhir2/narrative/expected/person-expected.html

```diff
@@ -1 +1 @@
-<div xmlns="http://www.w3.org/1999/xhtml"><table class="hapiPropertyTable"><tbody><tr><td>Id:</td><td>Person/example</td></tr><tr><td>Name:</td><td> Peter James <b>CHALMERS </b> , Jim </td></tr><tr><td>Gender:</td><td>MALE</td></tr><tr><td>Birth Date:</td><td>25/12/1974</td></tr><tr><td>Address:</td><td><span>534 Erewhon St </span><br/><span>PleasantVille </span><span>Vic </span></td></tr><tr><td>Telecom:</td><td> (HOME) , PHONE: <a href="tel:(03) 5555 6473">(03) 5555 6473</a> (WORK) , EMAIL: <a href="mailto:Jim@example.org">Jim@example.org</a> (HOME) </td></tr><tr><td>Identifier:</td><td><div>12345</div></td></tr><tr><td>Managing Organisation:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/Organization/example">Organization/example</a></td></tr><tr><td>Active:</td><td>true</td></tr><tr><td>Link:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/RelatedPerson/peter">Peter Chalmers</a> , <a href="http://localhost:8080/openmrs/ws/fhir2/R4/Patient/example">Peter Chalmers</a></td></tr></tbody></table></div>
+<div xmlns="http://www.w3.org/1999/xhtml"><table class="hapiPropertyTable"><tbody><tr><td>Id:</td><td>Person/example</td></tr><tr><td>Name:</td><td> Peter James <b>CHALMERS </b> , Jim </td></tr><tr><td>Gender:</td><td>MALE</td></tr><tr><td>Birth Date:</td><td>25/12/1974</td></tr><tr><td>Address:</td><td><span>534 Erewhon St </span><br/><span>PleasantVille </span><span>Vic </span></td></tr><tr><td>Telecom:</td><td> (HOME) , PHONE: <a href="tel:(03) 5555 6473">(03) 5555 6473</a> (WORK) , EMAIL: <a href="mailto:Jim@example.org">Jim@example.org</a> (HOME) </td></tr><tr><td>Identifier:</td><td><div>12345</div></td></tr><tr><td>Managing Organization:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/Organization/example">Organization/example</a></td></tr><tr><td>Active:</td><td>true</td></tr><tr><td>Link:</td><td><a href="http://localhost:8080/openmrs/ws/fhir2/R4/RelatedPerson/peter">Peter Chalmers</a> , <a href="http://localhost:8080/openmrs/ws/fhir2/R4/Patient/example">Peter Chalmers</a></td></tr></tbody></table></div>
```

Comment:
Changed `Organisation` to `Organization` since the property name had `z` instead of `s`; hence the change in expected narrative.